app/server.R

```diff
@@ -385,15 +394,20 @@ server <- function(input, output, session) {
         geom_point(aes(y = Reported_Incidence, color = "Reported_Incidence")) +
         geom_line(aes(y = Reported_As_Of_Incidence, color = "Reported_As_Of_Incidence")) +
         geom_point(aes(y = Reported_As_Of_Incidence, color = "Reported_As_Of_Incidence"))
-      if (input$showForecasts) {
-        finalPlot <- finalPlot +
-          geom_line(aes(y = Quantile_50, color = Forecaster)) +
-          geom_point(aes(y = Quantile_50, color = Forecaster, shape = Forecaster))
-      }
     } else {
       finalPlot <- finalPlot + geom_line(aes(y = Reported_Incidence)) +
         geom_point(aes(y = Reported_Incidence))
     }
+    if (input$showForecasts) {
+      maxLim <- max(
+        as.Date(input$asOf) + 7 * 4,
+        CURRENT_WEEK_END_DATE() + 7 * 1
```

Comment:
Does this mean we always display x-axis out to at least 1 week in the future, even when viewing as-ofs deep in the past?

Reply:
Switched this logic to using autoscaling if as-ofs are far enough in the past.